hbase-server/src/main/java/org/apache/hadoop/hbase/monitoring/MonitoredRPCHandlerImpl.java

```diff
@@ -231,6 +231,13 @@ public synchronized void markComplete(String status) {
     this.packet = null;
   }
 
+  @Override
+  public synchronized void cleanup() {
```

Comment:
We do not have this method before?

Reply:
Yes, it only used markComplete to release reference.

Reply:
So why not call markComplete here?

Reply:
Yes, the status is monitoring the RPC handler, so using markComplete is more properly.